src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -96,6 +96,8 @@ public class RestoreTask : Microsoft.Build.Utilities.Task, ICancelableTask, IDis
         /// </summary>
         public string EmbedFilesInBinlog { get; set; }
 
+        public string SDKAnalysisLevel { get; set; }
```

Comment:
```suggestion
        public string SdkAnalysisLevel { get; set; }
```

It's convention to have initialisms and acronyms only upper case the first letter, which is why you see APIs like `XmlReader`, not `XMLReader`, `JsonDocument`, not `JSONDocument`, `SqlClient`, not `SQLClient`, etc.  I recently had a disagreement with Nikolche about MSBuild, because I believe if we follow convention, it should be `MsBuild`. But I think this is the only potential exception.